kuma/core/tests/test_commands.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management import call_command, CommandError
+from django.utils.six import StringIO
+
+from nose.tools import eq_
+
+from kuma.core.tests import TestCase
+from kuma.users.models import User
+from kuma.users.tests import user
+
+
+class TestIHavePowerCommand(TestCase):
+    def test_help(self):
+        out = StringIO()
+        with self.assertRaises(CommandError) as cm:
+            call_command('ihavepower', stdout=out)
+
+        exc = cm.exception
+        eq_(exc.message, 'Error: too few arguments')
+
+    def test_user_doesnt_exist(self):
+        out = StringIO()
+        with self.assertRaises(CommandError) as cm:
+            call_command('ihavepower', 'fordprefect', stdout=out)
+
+        exc = cm.exception
+        eq_(exc.message, 'User fordprefect does not exist.')
+
+    def test_user_exists(self):
+        out = StringIO()
+        user(username='fordprefect', save=True)
+        call_command('ihavepower', 'fordprefect', stdout=out)
+
+        ford = User.objects.get(username='fordprefect')
```

Comment:
There is a `kuma.users.tests.UserTestCase` that provides a `self.user_model` test instance attribute that uses `django.contrib.auth.get_user_model` instead of hardcoding the use of our `User` class. I know that we won't likely change the User class, but since the swappable user model is a config value it would make the code brittle to not use the officially sanctioned way to fetch the user model.